<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slice_item_review</class>
 <widget class="QDialog" name="slice_item_review">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slice viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="slice_info_groupbox">
       <property name="title">
        <string>Slice information</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="topMargin">
         <number>1</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>full name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="full_name_lineEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Animal:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="animals_siv_comboBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Segment:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="segments_siv_comboBox">
            <property name="maxVisibleItems">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Slize number:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="slizeNumber_lineEdit"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Segment position:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="position_lineEdit"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Pict is Invert</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QCheckBox" name="isInvert_checkBox">
            <property name="text">
             <string/>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Comment</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QPlainTextEdit" name="comment_plainTextEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="animalType_siv_comboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Animal Type:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="markerType_siv_lineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Marker type:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="slice_image_label">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="new_experiment_pushButton">
         <property name="text">
          <string>New experiment</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="apply_changes_pushButton">
         <property name="text">
          <string>Apply changes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteSlice_siv_pushButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="process_selected_experiment_pushButton">
         <property name="text">
          <string>Process selected experiment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_selected_experiment_pushButton">
         <property name="text">
          <string>Delete selected experiment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="upload_selected_experiment_pushButton">
         <property name="text">
          <string>Upload selected experiment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_pushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Experiments list</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="neuDetectionResults_treeWidget">
          <attribute name="headerDefaultSectionSize">
           <number>180</number>
          </attribute>
          <column>
           <property name="text">
            <string>Cells group name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Neurons found</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>detection method</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>date done</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
